<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_leer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer archivos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_filtrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parametros (&quot;error&quot; o &quot;warning&quot; o &quot;INFO&quot;)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mostrar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar todos los logs</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_salir">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_filtrar_severidad">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">a</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_leer">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer archivos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_filtrar_severidad">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar (Escribir en el recuerdro los parametros)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mostrar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar todos los logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_router">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>521</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_filtrar_fecha">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar por fecha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_filtrar_fecha">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar por Fecha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejemplo de uso:       2024 -06-11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar por hora</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_filtrar_hora">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejemplo de uso:       10:25:48</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_filtrar_hora">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar por Hora</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
